### PR TITLE
op: add page

### PR DESCRIPTION
**Pull request by wangme88**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/op.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# op
+
+> An official command-line interface for 1Password's desktop app.
```
Comment (kbdharun) [inline pages/common/op.md:3]:
```suggestion
> Official CLI for 1Password's desktop app.
```

File: pages/common/op.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Sign in to a `1Password` account:
+
+`op signin`
+
+- List all vaults:
+
+`op vault list`
+
+- Print item detail in `JSON` format:
```
Comment (kbdharun) [inline pages/common/op.md:14]:
```suggestion
- Print item details in JSON format:
```

File: pages/common/op.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# op
+
+> An official command-line interface for 1Password's desktop app.
+> More information: <https://developer.1password.com/docs/cli>.
+
+- Sign in to a `1Password` account:
```
Comment (kbdharun) [inline pages/common/op.md:6]:
```suggestion
- Sign in to a 1Password account:
```

File: pages/common/op.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`op run -- {{command}}`
+
+- Pass secret references from an environment file to a command:
+
+`op run --env-file {{path/to/env_file.env}} -- {{command}}`
+
+- Read secret references from an input file and save plaintext secrets to an output file:
+
+`op inject -i {{path/to/file1}} -o {{path/to/file2}}`
```
Comment (kbdharun) [inline pages/common/op.md:36]:
```suggestion
`op inject -i {{path/to/input_file}} -o {{path/to/output_file}}`
```

File: pages/common/op.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- List all vaults:
+
+`op vault list`
+
+- Print item detail in `JSON` format:
+
+`op item get {{item_name}} --format json`
+
+- Create a new item with category in the default vault:
```
Comment (kbdharun) [inline pages/common/op.md:18]:
```suggestion
- Create a new item with a category in the default vault:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/op.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Print item details in JSON format:
+
+`op item get {{item_name}} --format json`
+
+- Create a new item with a category in the default vault:
+
+`op item create --category {{category_name}}`
+
+- Print a referenced secret in `stdout`:
```
Comment (acuteenvy) [inline pages/common/op.md:22]:
```suggestion
- Print a referenced secret to `stdout`:
```

File: pages/common/op.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Pass secret references from exported environment variables to a command:
+
+`op run -- {{command}}`
+
+- Pass secret references from an environment file to a command:
+
+`op run --env-file {{path/to/env_file.env}} -- {{command}}`
+
+- Read secret references from an input file and save plaintext secrets to an output file:
```
Comment (acuteenvy) [inline pages/common/op.md:34]:
```suggestion
- Read secret references from a file and save plaintext secrets to a file:
```

File: pages/common/op.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+`op inject -i {{path/to/input_file}} -o {{path/to/output_file}}`
```
Comment (acuteenvy) [inline pages/common/op.md:36]:
```suggestion
`op inject --in-file {{path/to/input_file}} --out-file {{path/to/output_file}}`
```

File: pages/common/op.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+# op
+
+> Official CLI for 1Password's desktop app.
+> More information: <https://developer.1password.com/docs/cli>.
```
Comment (acuteenvy) [inline pages/common/op.md:4]:
```suggestion
> More information: <https://developer.1password.com/docs/cli/reference>.
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**